make/crosscompile.jetson.mk

```diff
@@ -57,10 +57,10 @@ DEBUG = 0
 USE_SIGNAL_HANDLER = 1
 
 # the additional link flags you want to add
-ADD_LDFLAGS = -L${CROSS_ROOT}/lib
+ADD_LDFLAGS = -L${CROSS_ROOT}/lib -L/usr/lib/aarch64-linux-gnu/
```

Comment:
There can be 2 options: host libraries and cross compile libraries. By mixing both we might get in trouble. I suggest to find out why `${CROSS_ROOT}/lib` is not working (maybe try without `/lib`?)

Reply:
Yeah I like that thought. I believe you Pedro that you checked that, but I think it's still a good idea to have your investigation documented because I also struggled at finding the right headers. Describing your thoughts helps others to understand how and what you did and might also offer the possibility to get maybe improve the solution :)

Reply:
@lebeg As I said, CROSS_ROOT != /usr/lib/aarch64-linux-gnu/  I checked before, why do you want me to investigate again? /usr/lib/aarch64-linux-gnu/ is where nvidia packages go, there's no need to investigate further in my opinion.

Reply:
If you want to refactor the whole process please go ahead, this is outside of the scope of this fix.

Reply:
So where does cross root point to?

Reply:
>  /usr/lib/aarch64-linux-gnu/ is where nvidia packages go, there's no need to investigate further in my opinion.

I think that nvidia packages (if they are not in `${CROSS_ROOT}/lib`) should be installed or symlinked there instead.

Reply:
`/usr/lib/aarch64-linux-gnu/` is too generic (based on the name alone) to contain only CUDA libraries. Therefore it's confusing and is complicating the build in my opinion, which can lead to host/target library mixtures.

Reply:
That's a good point. I would suggest you send a PR a posteriori improving this. I think the Jetson & dockcross build is a bit hacky anyway. My PR fixes cross compilation, further refactor and improvements are out scope of this fix.

Reply:
@marcoabreu what should I document? tomorrow the nvidia packages will put the library in some other folder and this will break again and the comment will be outdated. You go into the container and look for the library there's not much to it.

Reply:
Maybe let's get @KellenSunderland input as he is more knowledgeable of Jetson and dockcross.